CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+* [FEATURE] Add -non-api-path-passthrough flag (disabled by default), which allows
+ exposing all non API resources from upstream without enforcing (e.g Prometheus UI).
```

Comment:
Let's add a note how this is safe in regards to for example the targets and configuration pages.